docs/en/latest/plugins/aws-lambda.md

```diff
@@ -0,0 +1,156 @@
+| function_uri      | string | required    |          |   | The aws api gateway endpoint which triggers the lambda serverless function code.   |
+| authorization   | object | optional    |         |     |  Authorization credentials to access the cloud function.                                                             |
+| authorization.apikey | string | optional    |             |     | Field inside _authorization_. The generate API Key to authorize requests to that endpoint of the aws gateway. |                         |
+| authorization.iam | object | optional    |             |     | Field inside _authorization_. AWS IAM role based authorization, performed via aws v4 request signing. See schema details below ([here](#iam-authorization-schema)). |                         |
+| timeout  | integer | optional    | 3000           | [100,...]     | Proxy request timeout in milliseconds.   |
+| ssl_verify  | boolean | optional    | true           | true/false     | If enabled performs SSL verification of the server.                     |
+| keepalive  | boolean | optional    | true           | true/false     | To reuse the same proxy connection in near future. Set to false to disable keepalives and immediately close the connection.                    |
+| keepalive_pool  | integer | optional    | 5          | [1,...]     | The maximum number of connections in the pool.              |
+| keepalive_timeout  | integer | optional    | 60000           | [1000,...]     |  The maximal idle timeout (ms).                     |
+
+### IAM Authorization Schema
+
+| Name             | Type   | Requirement  | Default        | Valid       | Description                                                                                |
+| -----------      | ------ | -----------  | -------        | -----       | ------------------------------------------------------------                               |
+| accesskey        | string | required     |                |             | Genereated  access key ID from aws IAM console.                                            |
+| secret_key       | string | required     |                |             | Genereated access key secret from aws IAM console.                                         |
+| aws_region       | string | optional     | "us-east-1"    |             | The aws region where the request is being sent.                                            |
+| service          | string | optional     | "execute-api"  |             | The service that is receiving the request (In case of Http Trigger it is "execute-api").   |
+
+## How To Enable
+
+The following is an example of how to enable the aws-lambda faas plugin for a specific route URI. Calling the apisix route uri will make an invocation to the lambda function uri (the new upstream).  We are assuming your cloud function is already up and running.
```

Comment:
Please use APISIX route instead of `apisix route`. Need to fix other similar places.

Reply:
We should use `APISIX` always

Reply:
Okay. Got it